PURPOSE.md

```diff
@@ -0,0 +1,21 @@
+## Why indigo-react? indigo-static? indigo-tokens?
+
+The prior indigo project iterations all lived in the same repository. This repository comprised react components, static styles, and design constants loosely organized, as well as the documentation, example sites, and build steps for each.
+
+One of the main problems with the prior iteration is that there was no process for publishing and using each of these libraries. Previously, developers would copy components from indigo into the project they were currently working on OR use whatever design component tools they were comfortable using. If it is faster to use a library you are comfortable with, like Tachyons, one might use that instead of indigo's static library.
+
+Projects are moving targets with changing requirements, so sometimes these components would need updates or tweaks. A developer might update their indigo components or classes, causing their version to be out of sync with the `master` copy, and also out of sync with all other developers.
+
+New, useful components that were created through project work were tough to integrate into the indigo project. If updates were made to indigo, one would have to update all the applications using indigo by hand, which is error prone and slow. There's also no mechanism to extend components in a sane manner that everyone can understand.
+
+## Updates
+
+1. Separate the projects – build tooling, publishing, and project structure are pretty different for static css, react component libraries, and design tokens. We have separated the projects. The common denominator is that they are all now part of the Javascript ecosystem (more below).
```

Comment:
We should also talk how it all works together. ie; all variables live inside indigo-tokens which is then read by indigo-react, etc

Reply:
Updated, let me know what you think of new version.